conf/routes

```diff
@@ -17,6 +17,7 @@ GET     /api2/atom/:id                  controllers.Api2.getAtom(id)
 PUT     /api2/atom/:id/asset            controllers.Api2.addAsset(id)
 
 GET     /api/youtube/categories         controllers.YoutubeCategory.list()
+PUT     /api/youtube/videos/:id/update  controllers.YoutubeVideos.update(id)
```

Comment:
maybe `/api/atom/:id/updateMetadata`? We'd still want to be able to update metadata for non-youtube atoms.